pep-0387.txt

```diff
@@ -78,6 +78,9 @@ be removed at any time in any way.  These include:
 - Test suites.  (Anything in the ``Lib/test`` directory or test
   subdirectories of packages.)
 
+Backward compatibility rules do not apply to any module or API that is
+explicitly documented as **Provisional**.
```

Comment:
(This should be another bullet presumably.)

Do you want to reference PEP 411 for the meaning of provisional?

Reply:
Both good points.  Fixed!